Android/PdfViewPager/source/PdfViewPager/PdfViewPager.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/2.1.2">
+  <PropertyGroup>
+    <TargetFrameworks>monoandroid9.0</TargetFrameworks>
+    <IsBindingProject>true</IsBindingProject>
+    <AssemblyName>PdfViewPager</AssemblyName>
+    <RootNamespace>PdfViewPager</RootNamespace>
+    <AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
+    <AndroidUseIntermediateDesignerFile>True</AndroidUseIntermediateDesignerFile>
+    <AndroidClassParser>class-parse</AndroidClassParser>
+    <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>PdfViewPager</PackageId>
```

Comment:
Xamarin prefix please

Reply:
Done, thanks for point out where the change should be 👍